README.md

```diff
@@ -8,23 +8,16 @@ The Supervisor is for now a node.js program, with a subset of its functionality
 
 We are using [waffle.io](https://waffle.io) to manage our tickets / issues, so if you want to track our progress or contribute take a look at [our board there](https://waffle.io/resin-io/resin-supervisor).
 
-## Running supervisor locally
+## Running a supervisor locally
 
-### Build a local supervisor image
-
-Build the supervisor with a specific repo and tag, e.g.
-```bash
-./tools/dev/dindctl build --image username/resin-supervisor:master --arch amd64
-```
-
-This will build the Supervisor docker image locally. If you then run `docker images` you should see the repo/tag you
-set there.
+This process will allow you to run a development instance of the supervisor on your local computer. It is not recommended for production scenarios, but allows someone developing on the supervisor to test changes quickly.
+The supervisor is run inside a docker-in-docker container that roughly emulates its environment when running on a ResinOS device.
 
 ### Set up config.json
 
-Add a `tools/dind/config.json` file from a staging device image. It should be configured for an x86 or amd64 device type so that you can push apps to it and they run properly on your computer.
+Add a `tools/dind/config.json` file from a staging or production device image. It should be configured for an x86 or amd64 device type (e.g. an Intel Nuc) so that you can push apps to it and they run properly on your computer.
 
-Note: don't use resinstaging for production devices. This is for development purposes only. A production (resin.io) config.json should work just as fine for this local supervisor, but we also don't recommend using this in production scenarios - ResinOS is way better suited for that.
+Note: don't use staging for production devices. This is for development purposes only. A production (resin.io) config.json should work just as fine for this local supervisor, but we also don't recommend using this in production scenarios - ResinOS is way better suited for that.
```

Comment:
Capitalize Don't and make resinOS lowercase. Also change "just as fine" to "just as well", and remove "way" before "better".